### PR TITLE
add registration test WINNF.FT.S.REG.20

### PR DESCRIPTION
**Pull request by janetliuboston**

### CODE REVIEW

File: src/harness/testcases/registration_testcase.py [hunk 1]
```diff
@@ -573,6 +573,49 @@ def test_WINFF_FT_S_REG_19(self):
...
+
+    # Inject FCC IDs
+    self._sas_admin.InjectFccId({'fccId': device_a['fccId']})
+    self._sas_admin.InjectFccId({'fccId': device_b['fccId']})
+    self._sas_admin.InjectFccId({'fccId': device_c['fccId']})
+
+    # Register devices
+    devices = [device_a, device_b, device_c]
+    request = {'registrationRequest': devices}
+    response = self._sas.Registration(request)
```
Comment (grao1991) [inline src/harness/testcases/registration_testcase.py]: try should include this line?

Comment (janetliuboston) [inline src/harness/testcases/registration_testcase.py]: Yes.

File: src/harness/testcases/registration_testcase.py [hunk 2]
```diff
@@ -542,6 +542,66 @@ def test_WINNF_FT_S_REG_13(self):
...
+         'fccId': device_d['fccId'],
+         'cbsdSerialNumber': device_d['cbsdSerialNumber'],
+         'airInterface': device_d['airInterface'],
+         'installationParam': device_d['installationParam']}
+    ]}
+    del conditionals_d['registrationData'][0]['installationParam']['antennaBeamwidth']
+    self._sas_admin.PreloadRegistrationData([conditionals_d])
+
+    # Inject FCC ID's
+    self._sas_admin.InjectFccId({'fccId': device_a['fccId']})
```
Comment (kate-harrison) [inline src/harness/testcases/registration_testcase.py]: Consider for compactness:

```
devices = [device_a, device_c, device_b, device_d]
for device in devices:
  self._sas_admin.InjectFccId({'fccId': device['fccId']})
  device['measCapability'] = []
```

Comment (janetliuboston) [inline src/harness/testcases/registration_testcase.py]: Agree. Already added in the later commit 0665 :)

Comment (kate-harrison) [inline src/harness/testcases/registration_testcase.py]: For some reason it's not showing up for me but I'll take your word for it :)

**Pull request merged**